unix/runner

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+"""
+Script to prepare JSON output for tidal sync servers from the list of hosts
+"""
+
+from __future__ import division
+
+import json
+
+from collections import defaultdict, namedtuple
+
+from ansible.executor.task_queue_manager import TaskQueueManager
+from ansible.inventory.manager import InventoryManager
+from ansible.parsing.dataloader import DataLoader
+from ansible.playbook.play import Play
+from ansible.plugins.callback import CallbackBase
+from ansible.vars.manager import VariableManager
+
+TOTAL_RESULTS = {'servers': []}
+
+def ram_allocated_gb(facts):
+    """Return total memory allocation in GB"""
+    return facts['ansible_memtotal_mb'] / 1024
+
+def ram_used_gb(facts):
+    """Return used memory in GB"""
+    return (facts['ansible_memtotal_mb'] - facts['ansible_memfree_mb']) / 1024
+
+def _size(key, mounts):
+    sizes = []
+    for item in mounts:
+        item = defaultdict(lambda: 0, item)
+        sizes.append(item[key])
+    return sum(sizes)
+
+def storage_allocated_gb(facts):
+    """Return total storage allocation in GB"""
+    return _size('size_total', facts['ansible_mounts']) / 1024**3
+
+def storage_used_gb(facts):
+    """Return used storage in GB"""
+    return (_size('size_total', facts['ansible_mounts']) -
+            _size('size_available', facts['ansible_mounts'])) / 1024**3
+
+def cpu_count(facts):
+    """Return the number of CPUs"""
+    proc = defaultdict(lambda: 0, facts)
+    return max([
+        int(proc['ansible_processor_count']),
+        int(proc['ansible_processor_vcpus'])
+    ])
```

Comment:
Same as above: you can use `get` here:

```python
def cpu_count(facts):
    """Return the number of CPUs"""
    return max([
        int(facts.get('ansible_processor_count', 0)),
        int(facts.get('ansible_processor_vcpus', 0))
    ])
```

Also, this might be a Python 3 thing, but I think you can just do `max(spam, eggs)` instead of `max([spam, eggs])`.